### PR TITLE
Migrate to viewbinding and kotlin gradle scripts

### PR DESCRIPTION
**Pull request by esensar**
- Replaced kotlinx sythetic with viewbinding
- Update build scripts to kotlin
- Intoduced version catalog, similar to Commons
- Updated kotlin from 1.7.10 to 1.9.0
- Updated Android Gradle Plugin to 8.1.0

### CODE REVIEW

File: app/src/main/kotlin/com/simplemobiletools/voicerecorder/activities/MainActivity.kt [hunk 1]
```diff
@@ -12,30 +12,33 @@ import com.simplemobiletools.commons.models.FAQItem
...
 import com.simplemobiletools.voicerecorder.extensions.config
 import com.simplemobiletools.voicerecorder.helpers.STOP_AMPLITUDE_UPDATE
 import com.simplemobiletools.voicerecorder.models.Events
 import com.simplemobiletools.voicerecorder.services.RecorderService
-import kotlinx.android.synthetic.main.activity_main.*
 import me.grantland.widget.AutofitHelper
 import org.greenrobot.eventbus.EventBus
 import org.greenrobot.eventbus.Subscribe
 import org.greenrobot.eventbus.ThreadMode
+import com.simplemobiletools.commons.R as CommonsR
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/voicerecorder/activities/MainActivity.kt:25]: wtf, where did you get this from...

File: app/src/main/kotlin/com/simplemobiletools/voicerecorder/adapters/ViewPagerAdapter.kt [hunk 2]
```diff
@@ -53,7 +53,7 @@ class ViewPagerAdapter(private val activity: SimpleActivity, val showRecycleBin:
     }
 
     fun finishActMode() {
-      (mFragments[1] as? PlayerFragment)?.finishActMode()
+        (mFragments[1] as? PlayerFragment)?.finishActMode()
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/voicerecorder/adapters/ViewPagerAdapter.kt:56]: why?

Comment (esensar) [reply app/src/main/kotlin/com/simplemobiletools/voicerecorder/adapters/ViewPagerAdapter.kt:56]: Formatting did this. Looks like this was not properly formatted originally

File: app/src/main/kotlin/com/simplemobiletools/voicerecorder/activities/MainActivity.kt (hunk 1 above)
Comment (esensar) [reply app/src/main/kotlin/com/simplemobiletools/voicerecorder/activities/MainActivity.kt]: Removed

File: app/src/main/kotlin/com/simplemobiletools/voicerecorder/adapters/ViewPagerAdapter.kt (hunk 2 above)
Comment (tibbi) [reply app/src/main/kotlin/com/simplemobiletools/voicerecorder/adapters/ViewPagerAdapter.kt:56]: right, my bad

**Pull request merged**